crux_http/tests/with_tester.rs

```diff
@@ -35,6 +35,7 @@ mod shared {
                 Event::Get => {
                     caps.http
                         .get("http://example.com")
+                        .header("Authorization", "secret-token")
```

Comment:
Wasn't sure whether to update in this file or in with_shell.rs? These ones look more current.

Reply:
yeah, `with_shell` will probably go at some point. Just there as an example really.